### PR TITLE
fix changelog for discussion of type requirement

### LINKED ISSUE #1867: Should type be mandatory for tiles and sizes in Image API 3.0?
The 3.0 change log says:

"1.2.4. Require the type property on all resources, with new values

The type property with a single value is now required on all resources, including content resources and services. This serves several purposes, including facilitating object mapping code libraries, and forcing the serialization to generate a JSON object for the resource, not just a string with the resource’s URI. The values for type have been changed to version-specific strings that avoid the namespace structure, for example from iiif:Image in 2.1 to ImageService3 in 3.0. Note that the @type property is used only when referring to object from older specifications such as the Authentication API 1.0." 

https://iiif.io/api/image/3.0/change-log/#124-require-the-type-property-on-all-resources-with-new-values 

But type on tiles and sizes is optional.

### PR DESCRIPTION
**Pull request by azaroth42**
Correct changelog and add links to issues regarding mandatory-ness of `type` in Image 3.0

Closes #1867

### CODE REVIEW

File: source/api/image/3.0/change-log.md [hunk 1]
```diff
@@ -68,9 +68,11 @@ The `license` property was renamed to `rights` to accommodate both rights statem
...
+The `type` property with a single value is now required at the top level of the info.json document, and any externally referenced resources including content resources such as further images and services. This serves several purposes, including facilitating object mapping code libraries, and forcing the serialization to generate a JSON object for the resource, not just a string with the resource's URI. The values for `type` have been changed to version-specific strings that avoid the namespace structure, for example from `iiif:Image` in 2.1 to `ImageService3` in 3.0. Note that the `@type` property is used only when referring to object from older specifications such as the Authentication API 1.0. The `type` property is optional for entries in the tiles (`Tile`) and sizes (`Size`) arrays.
+
+See issue [#1185](https://github.com/IIIF/api/issues/1185) and [#1867](https://github.com/IIIF/api/issues/1867)
```
Comment (zimeon) [inline source/api/image/3.0/change-log.md]: See issue**s** <-- plural

File: source/api/image/3.0/change-log.md [hunk 2]
```diff
@@ -68,9 +68,11 @@ The `license` property was renamed to `rights` to accommodate both rights statem
 
 See issues [#644](https://github.com/IIIF/api/issues/644) and [#1479](https://github.com/IIIF/api/issues/1479).
 
-#### 1.2.4. Require the `type` property on all resources, with new values
+#### 1.2.4. Require the `type` property on info.json and external references, with new values
```
Comment (zimeon) [inline source/api/image/3.0/change-log.md]: Should `info.json` be code font? Not a strong opinion but I lean that way

Comment (azaroth42) [reply source/api/image/3.0/change-log.md]: Not in a title, I think?

**Comment by azaroth42** [comment]
@tomcrane ?

File: source/api/image/3.0/change-log.md [hunk 3]
```diff
@@ -68,9 +68,11 @@ The `license` property was renamed to `rights` to accommodate both rights statem
...
+The `type` property with a single value is now required at the top level of the image information document (`info.json`), and any externally referenced resources including content resources such as further images and services. This serves several purposes, including facilitating object mapping code libraries, and forcing the serialization to generate a JSON object for the resource, not just a string with the resource's URI. The values for `type` have been changed to version-specific strings that avoid the namespace structure, for example from `iiif:Image` in 2.1 to `ImageService3` in 3.0. Note that the `@type` property is used only when referring to object from older specifications such as the Authentication API 1.0. The `type` property is optional for entries in the tiles (`Tile`) and sizes (`Size`) arrays.
+
```
Comment (tomcrane) [inline source/api/image/3.0/change-log.md]: > that the `@type` property is used only when referring to object from older specifications

objects?

Comment (zimeon) [reply source/api/image/3.0/change-log.md]: an object ?

**Comment by azaroth42** [comment]
Change made. @tomcrane can merge

**Pull request merged**